quic/s2n-quic-core/src/recovery/congestion_controller/fuzz_target.rs

```diff
@@ -0,0 +1,223 @@
+// Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+use crate::{
+    packet::number::PacketNumberSpace,
+    path::MINIMUM_MTU,
+    random,
+    recovery::{
+        bbr::BbrCongestionController, CongestionController, CubicCongestionController, RttEstimator,
+    },
+    time::{testing::Clock, Clock as _, Timestamp},
+};
+use bolero::{check, generator::*};
+use core::time::Duration;
+use std::collections::VecDeque;
+
+struct SentPacketInfo<PacketInfo> {
+    sent_bytes: u16,
+    time_sent: Timestamp,
+    cc_packet_info: PacketInfo,
+}
+
+#[derive(Debug, TypeGenerator)]
+enum Operation {
+    IncrementTime {
+        /// The milli-second value by which to increase the timestamp
+        millis: u16,
+    },
+    PacketSent {
+        #[generator(1..=255)]
+        count: u8,
+        #[generator(1200..=9000)]
+        bytes_sent: u16,
```

Comment:
Do we want to have fewer bytes sent that 1200? That's just the minimum for initial packets.

Reply:
Ah yep. It can even be 0 in the case of a non-congestion controlled packet